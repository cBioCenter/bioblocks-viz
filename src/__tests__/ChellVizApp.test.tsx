import { shallow } from 'enzyme';
import * as React from 'react';

import { ChellVizApp } from '~chell-viz~';

describe('ChellVizApp', () => {
  it('Should match existing snapshot when given no props.', () => {
    expect(shallow(<ChellVizApp />)).toMatchSnapshot();
  });
});
