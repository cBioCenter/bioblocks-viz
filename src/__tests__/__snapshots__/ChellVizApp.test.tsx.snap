// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChellVizApp Should match existing snapshot when given no props. 1`] = `
<div
  id="ChellVizApp"
>
  <Grid
    divided="vertically"
  >
    <GridRow>
      <VizPanelContainer
        allowUploads={false}
        dataDirs={
          Array [
            "assets/datasets/hpc/full",
          ]
        }
        initialVisualizations={
          Array [
            "Spring",
            "TENSOR-T-SNE",
          ]
        }
        numPanels={2}
        supportedVisualizations={
          Array [
            "Spring",
            "TENSOR-T-SNE",
          ]
        }
      />
    </GridRow>
    <AnatomogramContainer
      height="300px"
      style={Object {}}
      width="400px"
    />
    <GridRow
      centered={true}
      style={
        Object {
          "padding": "0 50px",
        }
      }
    >
      <IframeComm
        attributes={
          Object {
            "height": "500px",
            "src": "undefined/morpheus.html",
            "width": "100%",
          }
        }
        postMessageData={Object {}}
        serializeMessage={true}
        targetOrigin="*"
      />
    </GridRow>
  </Grid>
</div>
`;
